scripts/license/add.py

```diff
@@ -16,22 +16,29 @@
 Adds a license header to files.
 """
 
+import os
 from pathlib import Path
 import subprocess
 
 from license_header import license_header, has_license_header
 
-PROJECT_ROOT = Path(subprocess.check_output(
-    "git rev-parse --show-toplevel".split()).strip().decode("utf-8"))
+PROJECT_ROOT = subprocess.check_output(
+    "git rev-parse --show-toplevel".split()).strip().decode("utf-8")
+target_dirs = [os.path.join(PROJECT_ROOT, d) for d in os.listdir(PROJECT_ROOT)
+               if d != "vendor" and os.path.isdir(os.path.join(PROJECT_ROOT, d))]
```

Comment:
It may be a matter of taste, but using `pathlib` would make the code easier to read and write. Also, it would maintain consistency with other parts that are using `pathlib`.